CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [v1.8.1] - 2019-03-18
+
+- #214 add support for enable_proxy_protocol. - @mregmi
```

Comment:
version should be `1.9.0` per [CONTRIBUTING#versioning](https://github.com/digitalocean/godo/blob/master/CONTRIBUTING.md#versioning)

Reply:
done. thanks